core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -190,6 +253,9 @@ oppia.directive('conversationSkin', [function() {
         var hasInteractedAtLeastOnce = false;
         var _answerIsBeingProcessed = false;
         var _nextFocusLabel = null;
+        // This variable is used only when viewport is narrow.
+        // Indicates whether tutor card is active or supplemental card
```

Comment:
Replace this line with: "Indicates whether the tutor card is displayed."

(Note: the tutor card is, by definition, not the supplemental card.)